ArduPlane/mode_takeoff.cpp

```diff
@@ -68,59 +68,78 @@ bool ModeTakeoff::_enter()
 
 void ModeTakeoff::update()
 {
+    // nothing works if we dont have a valid position!
+    if (!plane.current_loc.initialised()) {
+        return;
```

Comment:
Still need to do something. Either:
```
        plane.calc_nav_roll();
        plane.calc_nav_pitch();	
        plane.calc_throttle();
```
Or set targets to 0.
```
nav_roll_cd = 0;
nav_pitch_cd = 0;
plane.calc_throttle();
```

If this is basically a no-op with no postilion having position should be a requirement checked in the `_enter` function.

Reply:
I guess if you lose position while flying you still want to update roll and pitch...so I just will enclose the waypoint setup stuff with the qualifier...